projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -53,6 +53,7 @@
 			'jetpack_relatedposts_enabled'            => '(bool) Enable related posts?',
 			'jetpack_relatedposts_show_headline'      => '(bool) Show headline in related posts?',
 			'jetpack_relatedposts_show_thumbnails'    => '(bool) Show thumbnails in related posts?',
+			'jetpack_relatedposts_append_to_posts'    => '(bool) Automattically display related posts to the bottom of each one of your posts?',
```

Comment:
Nit: Another extra `t`!
Also, another usage where I think "at" would be better than "to".
This is another place where it may be better to keep the sentence simpler, e.g.
`(bool) Automatically display related posts after each post?`
```suggestion
			'jetpack_relatedposts_append_to_posts'    => '(bool) Automatically display related posts at the bottom of each one of your posts?',
```

Also, this applies in all of the endpoint classes.

Reply:
Good point. And I always make that typo :)

I've updated all references in 8c0ced8298baab03abc66b87f851b70f7a8a3abc